hera_sim/adjustment.py

```diff
@@ -144,6 +144,8 @@ def adjust_to_reference(
     # Check if the target object is a Simulator, but work with a UVData object.
     target_is_simulator = isinstance(target, Simulator)
     target = _to_uvdata(target)
+    if not target.future_array_shapes:
```

Comment:
I think we can add `# pragma: nocover` here.

Reply:
yeah, good point. We'll be removing it anyway soon.